### ISSUE TITLE
Next moves for after meeting

### ISSUE THREAD

**Issue by rachelraso** [CMPUT301F21T04/Habitize#48]
<img width="540" alt="Screen Shot 2021-11-02 at 6 11 51 PM" src="https://user-images.githubusercontent.com/45080402/139968561-c1ad931c-e843-4d07-8461-1d524793def0.png">

**Comment by rpawliuk** [comment]
I would like to look more into the UML diagrams and examples of them

**Comment by ShanemelAsuncion** [comment]
I agree. We have to organize our UML into 3 diagrams to make it more readable!

**Comment by khalida1wwin** [comment]
great, we will do that. I will work on adding more tests

**Comment by rachelraso** [comment]
I am going to focus on separating the acticities from Behavioural ans structural classes in the UML :D

**Comment by rpawliuk** [comment]
I am going to finish up the progress bars this week
